shared/js/dialer/font_size_manager.js

```diff
@@ -91,13 +91,17 @@ var FontSizeManager = (function fontSizeManager() {
     if (infos && infos.overflow) {
       var overflowCount = FontSizeUtils.getOverflowCount(
         view.value || view.textContent, newFontSize, viewFont, viewWidth);
+
+      // Add two characters to the overflowcount to account for "…" width
+      overflowCount += 2;
       _useEllipsis(view, overflowCount, ellipsisSide);
+      view.dataset.ellipsedCharacters = overflowCount;
+    } else {
+      delete view.dataset.ellipsedCharacters;
```

Comment:
Please don't use ``delete``, set ``ellipsedCharacters`` to 0 or something else appropriate.